openfisca_france/model/prestations/aides_logement.py

```diff
@@ -24,6 +24,29 @@
 zone_apl_by_depcom = None
 
 
+class TypeEtatLogementFoyer(Enum):
+    non_renseigne = u"non renseigne"
+    logement_rehabilitee = u"Logement rehabilitee"
```

Comment:
```suggestion
    logement_rehabilite = u"Logement rehabilité"
```

(Et propager le nom de la variable `logement_rehabilite` avec un seul `e` ailleurs dans la PR)